lib/l10n/app_en.arb

```diff
@@ -270,5 +270,100 @@
   "createRegionAddBackGroundImageAcceptedFilesTitle": "Accepted file formats",
   "@createRegionAddBackGroundImageAcceptedFilesTitle": {},
   "createRegionAddBackGroundImageBoxTitle": "Add Picture",
-  "@createRegionAddBackGroundImageBoxTitle": {}
+  "@createRegionAddBackGroundImageBoxTitle": {},
+  "transferReceiveScanQRCode": "Scan QR Code",
+  "@transferReceiveScanQRCode": {},
+  "transferReceiveShareLink": "Share Link to Invoice",
+  "@transferReceiveShareLink": {},
+  "transferReceiveBalanceTotal": "Total",
+  "@transferReceiveBalanceTotal": {},
+  "transferReceiveMemo": "Memo: ",
```

Comment:
no we need to have memo 5 times? same with next and send
"transferReceiveMemo": "Memo: ",
"transferReceiveMemoFieldLabel": "Memo",
"transferReceiveMemoFieldHint": "Memo",
"transferSendAddNoteMemoLabel": "Memo",
"transferSendMemo": "Memo",

Reply:
Yeah, my primary concern was not to create tight coupling between a single string and multiple screens that may or may not use the same string in the same way 🤓 This avoids a scenario in the future when we tweak a string for one screen (e.g. `Memo` hint to `Add a note`) and suddenly the same string is changed everywhere 😅 

On second thought, though, I think some of these can be consolidated into generalized keys, but still separated by placement type. For example:
- transferReceiveMemo & transferSendMemo -> transferMemo
- transferReceiveMemoFieldLabel & transferSendAddNoteMemoLabel -> transferMemoFieldLabel

Will rework. Thanks for pointing it out! 🙏

Reply:
Generalized a coupla keys and fixed up duplicate keys 🤓